com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.Dilate.cs

```diff
@@ -112,7 +112,7 @@ public DataForDilation(ProbeReferenceVolume.Cell cell, float defaultThreshold)
 
                 for (int i = 0; i < probeCount; ++i)
                 {
-                    dilatedProbes[i].FromSphericalHarmonicsShaderConstants(cell.shBands, cell.shL0L1Data, cell.shL2Data, i);
+                    dilatedProbes[i].FromSphericalHarmonicsShaderConstants(cell.shBands, cell.state0.shL0L1Data, cell.state0.shL2Data, i);
```

Comment:
I would wrap this state0 into something more readable (something to make sure we know is the baking state)